qgispluginci/release.py

```diff
@@ -542,7 +542,7 @@ def release(
 
     release_tag = release_tag or release_version
 
-    if tx_api_token is not None:
+    if tx_api_token:
```

Comment:
It's almost equivalent no? Are you sure that will have the expected effect?

By the way, it's not really impacting so we can approve, merge and see.